CrazyCanvas/Include/Match/MatchBase.h

```diff
@@ -20,7 +20,7 @@ struct MatchDescription
 {
 	LambdaEngine::SHA256Hash LevelHash;
 	uint32 NumTeams = 2;
-	uint32 MaxScore = 1;
+	uint32 MaxScore = 100;
```

Comment:
we should keep it to 5 in master in my opinion. No reason to have 100. If you want to test out without win objective just increase is for testing purpose.